src/kubernetes.rs

```diff
@@ -213,6 +226,15 @@ impl<'a> Kubernetes<'a> {
             },
         );
 
+        let identity_key_path = config_dir.join("bootstrap-validator/identity.json");
+
+        secrets.insert(
+            "swqos".to_string(),
```

Comment:
What does this string mean? I see swqos.json used later as the bootstrap validator, so would something like "bootstrap" make more sense?

Reply:
ya the bootstrap has the most stake. and if we use the bootstrap for `--client-node-id`, we get much higher client TPS. but ya probably makes sense to change to bootstrap for clarity.